src/content/en/billions.markdown

```diff
@@ -0,0 +1,126 @@
+---
+layout: shared/root
+published: false
+description: "Building Web Apps for the Next Billion Users"
+title: "Building Web Apps for the Next Billion Users"
+translation_priority: 0
+---
+
+<style>
+.mdl-layout__content {
+    padding: 20px 0 0 100px;
+    }
+</style>
+
+<p class="intro">The next billion people coming online will be more diverse than ever, in every respect: physical location, cultural experience, device types, computer expertise and access to connectivity.
+<br><br>
+They will virtually all be mobile users.
+</p>
+
+This situation presents unique challenges for web developers, who must abandon the limited "mobile first" approach in favor of a comprehensive "mobile only" strategy.
+
+Developers can address these challenges by focusing on specific areas.
+
+
+## Connectivity
+
+### Challenge
+
+Many people have limited connectivity through geographical isolation or connection expense, and will regularly experience slow, intermittent, or no connectivity, either by necessity or by choice.
+
+### Solutions
+
+* Design for low-connectivity scenarios.
+* Reduce resource requests.
+* Plan for offline.
+* Make installs and updates as small and fast as possible.
+* Avoid load failures due to low bandwidth.
+
+[Learn more](.)
+
+
+## Content
+
+### Challenge
+
+Large-layout, graphics-heavy content frustrates users and discourages them from using your app. Content that is too wide or too tall is overlooked by some users and deliberately avoided by others.
+
+### Solutions
+
+* Design for small screens.
+* Downscale or eliminate images.
+* Make optional sections (navigation, sidebars, etc.) hidden but accessible.
+* Chunk long content into pages.
+* Embrace minimalist writing.
+
+[Learn more](.)
```

Comment:
Yes :). Dave and Meggin's idea is that each of these links to a 'discovery document' (which haven't been created yet). If necessary, I'll link to new/existing Web Fundamentals content instead.